CarbonQueryDatabase_Adapter/Versioning_32.json

```diff
@@ -0,0 +1,20 @@
+{
+  "Namespace": {
+    "ToNew": {
+      "BH.oM.CarbonQueryDatabase":  "BH.oM.Adapters.CarbonQueryDatabase",
+	  "BH.Engine.CarbonQueryDatabase":  "BH.Engine.Adapters.CarbonQueryDatabase"
```

Comment:
This is only needed in the individual projects - so no versioning for the Adapter is needed